src/ECS/Registry.cpp

```diff
@@ -9,8 +9,8 @@
 
 #include "Registry.h"
 
+#include "ECS/Systems/Implementations/RenderingSystem.h"
```

Comment:
Already in ECS and why is registry not using an interface?

Reply:
> I get moving implementations to a separate directory but why move the interfaces? They should be as easy access as possible.

I moved the interfaces to ECS/Systems

Reply:
> Already in ECS and why is registry not using an interface?

It should be. In fact, it is indirectly here.

Reply:
Made it use the interface directly